README.md

```diff
@@ -47,9 +47,11 @@ Must contain:
 - `optimizer`: optimizer used in training
   - `name`: Name of the optimizer (must)
   - `kwargs`: keyword arguments for the optimizer (can)
-- `pretrained_on`: list of datasets the model was trained on, valid entry:
-  - `name`: Name of dataset in the core library.
-  - TODO need optional ways of specifying identifiers for custom training data via url / doi
+- `pretrained_on`: description of pre-training:
+  - `datasets`: list of datasets the model was trained on, valid entry:
+    - `name`: Name of dataset in the core library.
+    - TODO need optional ways of specifying identifiers for custom training data via url / doi
+  - `n_epocs`: Number of epochs used in training (change to iterations ?)
```

Comment:
yes, actually `n_iterations` could be more accurate and is independent of dataset size (slightly biased by batch size though).
if we are keeping `n_epoch`s anyway, I'd suggest to not abbreviate away the `h`

Reply:
yeah, I will change to `n_iterations` later.
I was just to lazy to implement it in training, but I think it is better, because it says much more then the iteration count.